langtest/transform/representation.py

```diff
@@ -403,12 +399,16 @@ async def run(
         """
         progress = kwargs.get("progress_bar", False)
 
-        entity_representation = get_ethnicity_representation_dict(kwargs["raw_data"])
+        entity_representation = RepresentationOperation.get_ethnicity_representation_dict(
+            kwargs["raw_data"]
+        )
 
         for sample in sample_list:
             if sample.test_type == "min_ethnicity_name_representation_proportion":
-                entity_representation_proportion = get_entity_representation_proportions(
-                    entity_representation
+                entity_representation_proportion = (
+                    RepresentationOperation.get_entity_representation_proportions(
+                        entity_representation
+                    )
```

Comment:
I think there are extra parenthesis here and after

Reply:
it is due to black formatting issues